sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -733,33 +710,6 @@ def _add_argparse_args(cls, parser):
               'command.'))
 
 
-class FlinkOptions(PipelineOptions):
-
-  @classmethod
-  def _add_argparse_args(cls, parser):
-    parser.add_argument('--flink_master',
-                        type=str,
-                        help=
-                        ('Address of the Flink master where the pipeline '
-                         'should be executed. Can either be of the form '
-                         '\'host:port\' or one of the special values '
-                         '[local], [collection], or [auto].'))
```

Comment:
Do we want to replace this with a stub or something similar? Would be helpful to know that FlinkOptions are loaded from the runner.

Reply:
We might even keep some which are complemented by the DescribeRequest.

Reply:
They aren't required any longer; all runner options are now provided by the job server. This will hopefully avoid runner specific items sprinkled through SDKs. While we are familiar with these settings, they may even be confusing to users that work with a different runner and they really should not be flattened into a global namespace.

Reply:
Added stub.